layouts/partials/elements/r-benefits.html

```diff
@@ -0,0 +1,33 @@
+{{ if .benefitsEnabled }}
+
+{{- $announcements := partial "helpers/get-locale-data" "announcements" -}}
+
+{{ with $announcements.benefits }}
```

Comment:
This is not needed, since the announcements data is already available in the context. (As is evident by the if statement above.) So you can just remove line 3 and the with statement can be just `with .benefits`

Reply:
Fixed